build.gradle

```diff
@@ -1161,126 +1190,6 @@ subprojects {
 
 assemble.dependsOn(':checker:assembleForJavac')
 
-task checkBasicStyle(group: 'Format') {
-    description 'Check basic style guidelines, mostly whitespace.  Not related to Checkstyle tool.'
-    String[] ignoreDirectories = [
-        '.git',
-        '.gradle',
-        '.html-tools',
-        '.idea',
-        '.plume-scripts',
-        'annotated',
-        'api',
-        'plume-bib',
-        'bootstrap',
-        'build',
-        'jdk'
-    ]
-
-    String[] ignoreFilePatterns = [
-        '*.aux',
-        '*.bib',
-        '*.class',
-        '*.dvi',
-        '*.expected',
-        '*.gif',
-        '*.jar',
-        '*.jtr',
-        '*.log',
-        '*.out',
-        '*.patch',
-        '*.pdf',
-        '*.png',
-        '*.sty',
-        '*.toc',
-        '*.xcf',
-        '*~',
-        '#*#',
-        'CFLogo.ai',
-        'logfile.log.rec.index',
-        'manual.html',
-        'manual.html-e',
-        'junit.*.properties',
-        'securerandom.*',
-        'checker/dist/META-INF/maven/org.apache.bcel/bcel/pom.xml',
-        'checker/dist/META-INF/maven/org.apache.commons/commons-text/pom.xml',
-        'checker/nullness/junit-assertions.astub',
-        'framework/src/main/resources/git.properties'
-    ]
-
-    doLast {
-        FileTree tree = fileTree(dir: projectDir)
-        for (String dir : ignoreDirectories) {
-            tree.exclude "**/${dir}/**"
-        }
-        for (String file : ignoreFilePatterns) {
-            tree.exclude "**/${file}"
-        }
-        boolean failed = false
-        tree.visit {
-            if (!it.file.isDirectory()) {
-                boolean blankLineAtEnd = false
-                String fileName = it.file.getName()
-                boolean checkTabs = !fileName.equals('Makefile')
-                it.file.eachLine { line ->
-                    if (line.endsWith(' ')) {
-                        println("Trailing whitespace: ${it.file.absolutePath}")
-                        failed = true
-                    }
-                    if (checkTabs && line.contains('\t')) {
-                        println("Contains tab (use spaces): ${it.file.absolutePath}")
-                        failed = true
-                        checkTabs = false
-                    }
-                    if (!line.startsWith('\\') &&
-                            (line.matches('^.* (else|finally|try)\\{}.*$')
-                            || line.matches('^.*}(catch|else|finally) .*$')
-                            || line.matches('^.* (catch|for|if|while)\\('))) {
-                        // This runs on non-java files, too.
-                        println("Missing space: ${it.file.absolutePath}")
-                        failed = true
-                    }
-                    if (line.matches('^.*/\\* package-private \\*/.*$')) {
```

Comment:
Spotless doesn't seem to perform this check.
Instead of adding the basic checks back, I've [added a note](https://github.com/eisop/checker-framework/pull/472/commits/4fbcd52028d85b397adcd9898914713904d8116d).